common-moonmage.lic

```diff
@@ -120,4 +120,18 @@ module DRCMM
 
     store_div_tool(tool_storage)
   end
+
+  # There are many variants of a summoned moon weapon (blade, staff, sword, etc)
+  # This function checks if you're holding one then tries to wear it.
+  # https://elanthipedia.play.net/Shape_Moonblade
+  def wear_moon_weapon
+    moon_weapon_regex = /moon[\w]+/
+    moon_wear_messages = ['telekinetic']
+    if left_hand =~ moon_weapon_regex
+      bput("wear #{left_hand}", *moon_wear_messages)
+    end
+    if right_hand =~ moon_weapon_regex
+      bput("wear #{right_hand}", *moon_wear_messages)
+    end
+  end
```

Comment:
According to https://elanthipedia.play.net/Shape_Moonblade, all moon weapons are a single word that start with "moon". Such as "moonblade", "moonkatar", "moonstaff", etc. Therefore the regex at line 128 matches on the word "moon" followed by other characters.